MdeModulePkg/MdeModulePkg.dec

```diff
@@ -1187,6 +1187,9 @@
   # @Prompt Delay access XHCI register after it issues HCRST (us)
   gEfiMdeModulePkgTokenSpaceGuid.PcdDelayXhciHCReset|2000|UINT16|0x30001060
 
+  # Support alternative hardware queue sizes in NVME driver

```

Comment:
Add @Prompt and description of TRUE and FALSE setting and the default value.

Also, is this PCD really required?  What would be the impact if just assuming this is always enabled.  Prefer to not add new PCDs unless absolutely required.